<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Checkvaccinestatusscreen</class>
 <widget class="QDialog" name="Checkvaccinestatusscreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Co-vix</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(&quot;:/Co-vix Images/Co-vix logo header.png&quot;);
 background-repeat: no-repeat;
border-style: none;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>1161</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #ecf5fa;
border-radius: 20px ;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>1011</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>701</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
border-bottom: 2px solid #F47723;
border-radius: 1px ;
color: Black;
font-size: 14px;
letter-spacing: 4px;
 </string>
     </property>
     <property name="placeholderText">
      <string>Check Vaccine Status </string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>50</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(&quot;:/Co-vix Images/serchicon.png&quot;);
 background-repeat: no-repeat;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="checkButton">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>10</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(&quot;:/Co-vix Images/CHECK btn.png&quot;);
 background-repeat: no-repeat;
border-style: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>500</width>
      <height>85</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(&quot;:/Co-vix Images/CHECK VACCINE  STATUS LBL.png&quot;);
 background-repeat: no-repeat;
border-style: none;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>10</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-image: url(&quot;:/Co-vix Images/BACK Btn.png&quot;);
border: 0px solid #555555;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="nameLbl">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>380</y>
     <width>1051</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: Black;
font-size: 16px;
letter-spacing: 4px;</string>
   </property>
   <property name="text">
    <string>01. Name : </string>
   </property>
  </widget>
  <widget class="QLabel" name="firstdosLbl">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>440</y>
     <width>1051</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: Black;
font-size: 16px;
letter-spacing: 4px;</string>
   </property>
   <property name="text">
    <string>02. Date Of First Dose of Covid Vaccine And Vaccine Type : </string>
   </property>
  </widget>
  <widget class="QLabel" name="sendoseLbl">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>500</y>
     <width>1051</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: Black;
font-size: 16px;
letter-spacing: 4px;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;03. Date Of Second Dose of Covid Vaccine And Vaccine Type :  &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="thirddoseLbl">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>560</y>
     <width>1051</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: Black;
font-size: 16px;
letter-spacing: 4px;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;04. Date Of Third Dose of Covid Vaccine And Vaccine Type :  &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>620</y>
     <width>102</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(&quot;:/Co-vix Images/CHECK CLEAR btn.png&quot;);
background-repeat: no-repeat;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="errframe">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>1081</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(&quot;:/Co-vix Images/Sorry art.png&quot;);
 background-repeat: no-repeat;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="retryButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>250</y>
      <width>102</width>
      <height>42</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(&quot;:/Co-vix Images/RE-TRY btn.png&quot;);
background-repeat: no-repeat;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="openframe">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>1151</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(&quot;:/Co-vix Images/serart.png&quot;);
 background-repeat: no-repeat;
border-style: none;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_6">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>-30</y>
     <width>501</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  border: 2px solid #3867A9;
  border-radius: 25px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="signinBTN">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
background-image: url(&quot;:/Co-vix Images/LOG IN BTN.png&quot;);
border: 0px solid #555555;
border-style: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="homeBTN">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
background-image: url(&quot;:/Co-vix Images/HOME BTN.png&quot;);
border: 0px solid #555555;
border-style: none;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="aboutusBTN">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
background-image: url(&quot;:/Co-vix Images/ABOUT US  BTN.png&quot;);
border: 0px solid #555555;
border-style: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <zorder>frame_6</zorder>
  <zorder>frame</zorder>
  <zorder>frame_2</zorder>
  <zorder>nameLbl</zorder>
  <zorder>firstdosLbl</zorder>
  <zorder>sendoseLbl</zorder>
  <zorder>thirddoseLbl</zorder>
  <zorder>clearButton</zorder>
  <zorder>errframe</zorder>
  <zorder>openframe</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
